etc/.jscpd.json

```diff
@@ -0,0 +1,10 @@
+{
+    "threshold": 0,
+    "reporters": [
+        "consoleFull"
+    ],
+    "ignore": [
+        "**/workflows/**"
```

Comment:
Very generic, why wouldn't we say .GitHub/workflows/** ?

Reply:
@JonZeolla re-opening this because the suggestion didnt work, i have modified it to reflect whats working